### PR TITLE
just saying

### PR DESCRIPTION
**Pull request by bmdito**
great work!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # project-demo
 
-Hi. 
\ No newline at end of file
+Hi. 
+
```
Comment (bmdito) [inline README.md:4]: change this to hello

**Comment by belindanovoa** [comment]
Should of said Hello.

**Review by belindanovoa — APPROVED** [review]
wtf is going on

**Review by bmdito — COMMENTED** [review]
WALALA

**Pull request closed without merging**